narayana/ArjunaJTA/jta/tests/classes/org/jboss/narayana/rts/TxnTest.java

```diff
@@ -66,25 +62,26 @@ public static void tearDown() throws IOException {
 
     @Benchmark
     public void testTxn(Blackhole bh) throws IOException {
-        bh.consume(preExeTransaction());
-        // we want 2 txn participants so make 2 transactional requests
-        bh.consume(sendRequest());
-        bh.consume(TxnHelper.endTxn(txnClient, links));
+        bh.consume(testTxn());
     }
 
-    private boolean preExeTransaction() throws IOException {
-        links = TxnHelper.beginTxn(txnClient, TXN_URL);
+    public boolean testTxn() throws IOException {
```

Comment:
I have a similar comment here to [my previous one](https://github.com/jbosstm/performance/pull/155/files#r1068138022).